packages/smooth_app/lib/pages/preferences/country_selector/country_selector_provider.dart

```diff
@@ -47,146 +48,51 @@ class _CountrySelectorProvider extends PreferencesSelectorProvider<Country> {
   }
 
   @override
-  Future<List<Country>> onLoadValues() async {
-    final List<Country> localizedCountries = await CountriesHelper.getCountries(
-          userAppLanguageCode,
-        ) ??
-        <Country>[];
-
-    final List<Country> countries = await compute(
-      _reformatCountries,
-      (localizedCountries, userCountryCode),
-    );
-
+  Future<List<OpenFoodFactsCountry>> onLoadValues() async {
```

Comment:
Please keep the same methods with the same names here. I'm a bit lost in that code and I'm afraid you may skip interesting parts while trying to simplify the code.

Reply:
ping